docs/se/cors.adoc

```diff
@@ -109,41 +109,28 @@ the `CorsSupport` API to influence the xref:{rootdir}/se/webserver.adoc#routing[
 thereby determining how that resource is shared. (If desired, you can use <<Configuration, configuration>> instead
 of the low-level API.)
 
-The following code shows how to prepare your application's routing to support metrics and health support, as well as
-CORS.
+The following code shows one way to prepare your application's routing to support CORS.
 
 [[intro-quick-start-code-example]]
 [source,java]
 ----
-private static Routing createRouting(Config config) {
-    MetricsSupport metrics = MetricsSupport.create();
-    GreetService greetService = new GreetService(config);
-    HealthSupport health = HealthSupport.builder()
-            .addLiveness(HealthChecks.healthChecks())   // Adds a convenient set of checks
-            .build();
+static void routing(HttpRouting.Builder routing) {
     CorsSupport corsSupport = CorsSupport.builder()  // <1>
             .addCrossOriginConfig(CrossOriginConfig.builder() // <2>
```

Comment:
Should this be `addCrossOrigin` not `addCrossOriginConfig` (same for line 123)

Reply:
Absolutely. Fix coming. Thanks for catching that.